### PR TITLE
Using OC_ERR return val for OC failures

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

**Comment by zmiklank** [comment]
[test-all]

File: test/run-openshift-remote-cluster [hunk 1]
```diff
@@ -24,13 +24,13 @@ ct_os_set_ocp4
 
 trap ct_os_cleanup EXIT SIGINT
 
-ct_os_check_compulsory_vars
+ct_os_check_compulsory_vars || exit $OC_ERR
```
Comment (frenzymadness) [inline test/run-openshift-remote-cluster:27]: Where is the new variable defined?

**Review by frenzymadness — APPROVED** [review]
Just one question from me but feel free to ignore it. You understand this much more than me and I don't see anything wrong.

File: test/run-openshift-remote-cluster (hunk 1 above)
Comment (zmiklank) [reply test/run-openshift-remote-cluster:27]: It has been added to `test-lib-remote-openshift.sh` recently:

https://github.com/sclorg/container-common-scripts/commit/85f06416bf48f5c66766e605047d81e97d73199f

**Pull request merged**